pkg/manifests/config.go

```diff
@@ -202,31 +213,6 @@ func (u *UserWorkloadConfiguration) checkThanosRulerEvaluationInterval() error {
 	return nil
 }
 
-func (u *UserWorkloadConfiguration) checkAlertmanagerVersion() error {
```

Comment:
if we could have "warning propagation" and "4.19 alertmanager code cleaning" in two separate it'd be great.
I can help with babysitting.